storage/src/main/java/tech/pegasys/teku/storage/server/kvstore/BlindedBlockKvStoreDatabase.java

```diff
@@ -47,13 +47,16 @@
         CombinedUpdaterBlinded,
         HotUpdaterBlinded,
         FinalizedUpdaterBlinded> {
+  final BlindedHotBlockMigration<?> migrater;
 
   BlindedBlockKvStoreDatabase(
       final KvStoreCombinedDaoBlinded dao,
+      final BlindedHotBlockMigration<?> hotBlockMigrater,
```

Comment:
nit: Can rename the param to migrater. And I think it should be migrator  (browser spell check agrees)